### PR TITLE
users controller: summary and data pages.

### PR DESCRIPTION
**Pull request by artur79**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  end
+
+  def data; end
+
+  private
+
+  def find_user_group_team
+    # assume that user has only one team user group
+    @user_group = current_user.user_groups.joins(:group_type).where("group_types.name = 'team'").first
+  end
```
Comment (barbarawuk) [inline app/controllers/users_controller.rb:28]: We don't use @user_group anywhere in the user views. I get that it's needed for the TeamIndividualPresenter but perhaps a bit of refactoring is needed, as these pages shouldn't depend on the user being in a group.
(Also if they aren't in a group then end up with a nil:NilClass error.

Comment (artur79) [reply app/controllers/users_controller.rb:28]: so we can not assume that:
1. user always belongs to at least one user group
2. user has always team (user group type team) set ?

Comment (barbarawuk) [reply app/controllers/users_controller.rb:28]: I don't think we can assume either of those things. 
A smaller organisation might use Murfin+ and not bother with any organisational structure if there are e.g. less than 100 users.  And we can't assume they have a group type with 'team' as the name either. The groups types could be anything, directorate, division, specialty etc.

Comment (artur79) [reply app/controllers/users_controller.rb:28]: Make sense, I will address those assumptions in this PR next

**Comment by artur79** [comment]
@barbarawuk applied updates

**Pull request merged**